Sprint 2/DBConnection.java

```diff
@@ -0,0 +1,39 @@
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class DBConnection {
+	 private Connection connect = null;
+	 private Statement statement = null;
+	 private PreparedStatement preparedStatement = null;
+	 private ResultSet resultSet = null;
+	public Connection connect_func() throws SQLException {
+		
+	    if (connect == null || connect.isClosed()) {
+	        try {
+	            Class.forName("com.mysql.jdbc.Driver");
+	            
+	        } catch (ClassNotFoundException e) {
+	            throw new SQLException(e);
+	        }
+	        
+	        connect = (Connection) DriverManager

```

Comment:
I think that this line of code could be made a bit more readable if you have the (Connection) DriverManager and the .getConnection method on its own separate line.